Add more spec_urls; set some standard_tracks to false (APIs A-E)

Elchi3 opened the pull request:
This is a bit of a test PR to see if we could add spec_urls everywhere (especially for behavioral sub features). 
Some things came up where we should probably agree on spec_url consistency for a few common sub features:
- worker_support
- iterable sub features (forEach, keys, values, iterator, etc)
- secure_context_required 
I decided to use the spec_url for the interface itself, happy to hear other thoughts. 

Also, I think when a sub aspect of e.g. a method is described then it is okay to link to the same spec section as the method. So, basically the sub feature gets the same spec_url as the parent feature. 

I marked standard_track false for a few things: CSSPrimitiveValue, CSSValue, CSSValueList, DOMError, FeaturePolicy. Doesn't look like these will be going anywhere.

Fixes https://github.com/mdn/browser-compat-data/issues/27456.

Elchi3 committed:
Add more spec_urls; set some standard_tracks to false (APIs A-C)

caugner commented:
Let's mark as draft for now then, I'll happily go through the changes to double check once complete.

Elchi3 committed:
More specs

Elchi3 committed:
Merge branch 'main' into moar-spec-urls

caugner committed:
Merge branch 'main' into moar-spec-urls

caugner committed:
Replace geometry spec URLs

Mostly:

```diff
-https://drafts.fxtf.org/geometry/
+https://drafts.csswg.org/geometry/
```

caugner committed:
Merge branch 'main' into moar-spec-urls

Elchi3 commented:
I think we can close this PR. Claas merged my work into https://github.com/mdn/browser-compat-data/pull/29223 and that was merged.
